packages/mui-utils/src/useId.ts

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 
 let globalId = 0;
-export default function useId(idOverride?: string): string | undefined {
+function useRandomId(idOverride?: string): string | undefined {
```

Comment:
nit: it's not really random

Reply:
It's not random at all to be specific. This was just an old rebase. Will adjust.